src/pkynetics/data_import/dilatometry_importer.py

```
@@ -0,0 +1,89 @@
+                df["differential_change"].values, dtype=np.float64
+            ),
+        }
+
+        return result_data
+
+    except FileNotFoundError:
+        logger.error(f"File not found: {file_path}")
+        raise
+    except Exception as e:
+        logger.error(f"Error importing dilatometry data: {str(e)}")
+        raise ValueError(f"Unable to import dilatometry data. Error: {str(e)}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)